dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/GoverProportional.mo

```diff
@@ -1,17 +1,18 @@
 within Dynawo.Electrical.Controls.Machines.Governors;
 
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```

Comment:
Do we have to update the copyright of already existing models ?

Reply:
I don't think so actually so I came back to the old copyright